lisa_commands/commands.py

```diff
@@ -286,7 +292,7 @@ def kill_me(_query, slack_event):
             'By order of the SDF Armed Forces you have been killed',
             'Ask not for whom the bell tolls. It tolls for thee ~ John Donne',
```

Comment:
Huh, I always thought the Bell Tolls quote was Hemingway

Reply:
http://www.yourdailypoem.com/listpoem.jsp?poem_id=2118

I checked

Reply:
TIL